$OpenBSD: patch-cfg_cpp,v 1.1.4.1 2010/08/24 11:43:29 jasper Exp $

Remove './' from PATH.

Some windowmanagers inherent this PATH and that introduces some security
risk because './' can allow local attackers to trick a user into
accidentally executing code.


--- cfg.cpp.orig	Tue Aug 24 13:02:26 2010
+++ cfg.cpp	Tue Aug 24 13:02:51 2010
@@ -28,7 +28,7 @@ Cfg::Cfg() 
     : currentSession(-1)
 {
     // Configuration options
-    options.insert(option("default_path","./:/bin:/usr/bin:/usr/local/bin"));
+    options.insert(option("default_path","/bin:/usr/bin:/usr/local/bin"));
     options.insert(option("default_xserver","/usr/bin/X"));
     options.insert(option("xserver_arguments",""));
     options.insert(option("numlock",""));
